### PR TITLE
updated to Flink v0.10.1

### PR DESCRIPTION
**Pull request by romeokienzler**
updated to Flink v0.10.1 and tested if the web front-end comes up

### CODE REVIEW

File: flink-contrib/docker-flink/flink/Dockerfile [hunk 1]
```diff
@@ -25,8 +25,8 @@ RUN cat ~/.ssh/id_rsa.pub >> ~/.ssh/authorized_keys && chmod 600 ~/.ssh/*
 ##Flink 0.10.0 Installation
 ###Download:
 RUN mkdir ~/downloads && cd ~/downloads && \
-    wget -q -O - http://apache.spd.co.il/flink/flink-0.10.0/flink-0.10.0-bin-hadoop27-scala_2.11.tgz | tar -zxvf - -C /usr/local/
-RUN cd /usr/local && ln -s ./flink-0.10.0 flink
+    wget -q -O - http://mirror.switch.ch/mirror/apache/dist/flink/flink-0.10.1/flink-0.10.1-bin-hadoop27-scala_2.11.tgz| tar -zxvf - -C /usr/local/
```
Comment (zentol) [inline flink-contrib/docker-flink/flink/Dockerfile]: why did you change the address?

Comment (romeokienzler) [inline flink-contrib/docker-flink/flink/Dockerfile]: because switch is a very good and fast mirror

**Comment by hsaputra** [comment]
Do you have JIRA ticket filed for this? As part of ASF contribution please do file JIRA [1] and associate with the PR [2]. Thanks!

[1] https://issues.apache.org/jira/browse/FLINK/
[2] https://flink.apache.org/contribute-code.html

File: flink-contrib/docker-flink/flink/Dockerfile (hunk 1 above)
Comment (mxm) [inline flink-contrib/docker-flink/flink/Dockerfile]: I think it's fine to switch the mirror. There was no particular reason for the mirror before. It would be neat to go through the mirrors list at http://www.apache.org/dyn/closer.lua/flink/flink-0.10.1/flink-0.10.1-bin-hadoop27-scala_2.11.tgz but not necessary to change that now.

**Comment by mxm** [comment]
Thanks @hsaputra for the reminder. I've filed a JIRA: https://issues.apache.org/jira/browse/FLINK-3155

Looks good to merge otherwise.
